build.sbt

```diff
@@ -327,6 +327,16 @@ lazy val zincScripted = (project in internalPath / "zinc-scripted").
   ).
   configure(addSbtUtilScripted)
 
+lazy val crossTestBridges = {
+  Command.args("crossTestBridges", "<version>") { (state, args) =>
+    require(args.nonEmpty, "Missing Scala version argument.")
+    compilerBridgeScalaVersions.flatMap { (bridgeVersion: String) =>
+      s"wow $bridgeVersion" ::
+        s"${compilerBridge.id}/test" :: Nil
```

Comment:
No issue was opened against sbt/sbt-doge about the issue that triggered the change in #211.

So there's no clear information about whether there is an underlying issue in plz.